Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -157,15 +158,15 @@ private void OnPlayersSpawning(RulePlayerSpawningEvent ev)
             switch (i)
             {
                 case 0:
-                    name = $"Commander";
+                    name = $"Commander " + _random.Pick(_prototypeManager.Index<DatasetPrototype>("syndicate_names_elite").Values);
                     gear = commanderGear;
                     break;
                 case 1:
-                    name = $"Operator #{i}";
+                    name = $"Agent " + _random.Pick(_prototypeManager.Index<DatasetPrototype>("syndicate_names_elite").Values);
                     gear = medicGear;
                     break;
                 default:
-                    name = $"Operator #{i}";
+                    name = $"Operator " + _random.Pick(_prototypeManager.Index<DatasetPrototype>("syndicate_names_normal").Values); ;
```

Comment:
You should probably use lists + take and pick because you might get duplicates.

Reply:
Think I did this the way you wanted.